main.nf

```diff
@@ -448,6 +451,16 @@ if (!params.input && params.sentieon) {
         case 'annotate': break
         default: exit 1, "Unknown step ${step}"
     }
+} else if (!params.input && !params.sentieon && params.skip_markduplicates) {
+    switch (step) {
+        case 'mapping': break
+        case 'preparerecalibration': tsvPath = "${params.outdir}/Preprocessing/TSV/mapped.tsv"; break
+        case 'recalibrate': tsvPath = "${params.outdir}/Preprocessing/TSV/mapped_no_duplicates_marked.tsv"; break
+        case 'variantcalling': tsvPath = "${params.outdir}/Preprocessing/TSV/recalibrated.tsv"; break
+        case 'controlfreec': tsvPath = "${params.outdir}/VariantCalling/TSV/control-freec_mpileup.tsv"; break
```

Comment:
It looks good but maybe then it should be included in the `input.md` file that you can also start from the `preparerecalibration` and the `controlfreec`step and how to prepare this tsv?

Reply:
Ah never mind, I saw you already included these!!